websites/K/Keep Talking and Nobody Explodes/presence.ts

```diff
@@ -1,159 +1,240 @@
-          browsingTimestamp ??= Math.floor(Date.now() / 1000);
-          presenceData.startTimestamp = browsingTimestamp;
-
-          presenceData.smallImageKey = "reading";
-          presenceData.smallImageText = (await strings).reading;
-
-          presenceData.details = "Bomb Defusal Manual";
-          break;
-        //How to Play
-        case "how-to-play-pc.html":
-        case "how-to-play-mobile.html":
-        case "how-to-play-switch.html":
-        case "how-to-play-xbox.html":
-        case "how-to-play-playstation.html":
-        case "how-to-play-vr.html":
-        case "how-to-play-psvr.html":
-        case "how-to-play-gear-vr.html":
-        case "how-to-play-oculus-go.html":
-        case "how-to-play-oculus-quest.html":
-        case "how-to-play-daydream.html": {
-          browsingTimestamp = 0;
-          let platform: string;
-
-          presenceData.details = "How to Play";
-          switch (path[0].replace("how-to-play-", "").replace(".html", "")) {
-            case "pc":
-              platform = "PC/Mac/Linux";
-              break;
-            case "mobile":
-              platform = "iOS/Android";
-              break;
-            case "switch":
-              platform = "Nintendo Switch™";
-              break;
-            case "xbox":
-              platform = "Xbox One";
-              break;
-            case "playstation":
-              platform = "PlayStation®4";
-              break;
-            case "vr":
-              platform = "Oculus Rift/HTC Vive";
-              break;
-            case "psvr":
-              platform = "PlayStation®VR";
-              break;
-            case "gear-vr":
-              platform = "Samsung Gear VR";
-              break;
-            case "oculus-go":
-              platform = "Oculus Go";
-              break;
-            case "oculus-quest":
-              platform = "Oculus Quest";
-              break;
-            case "daydream":
-              platform = "Daydream";
-              break;
-            default:
-              break;
-          }
-          if (platform) presenceData.state = `on ${platform}`;
-          break;
-        }
-        //Other Languages
-        case "other-languages.html":
-          browsingTimestamp = 0;
-
-          presenceData.details = "Other Languages";
-          break;
-        //Unknown
-        case "index.html":
-          presence.setActivity();
-          return;
-        default:
-          presence.setActivity();
-          return;
-      }
-      break;
+  if (page === 0) {
+    presenceData.state =
+      document.querySelector<HTMLHeadingElement>(
+        ".versioning"
+      ).firstChild.textContent;
+  } else {
+    presenceData.state = `${(await strings).page} ${page + 1} / ${
+      pages.length
+    }: ${pages[page].children[0].children[1].textContent}`;
   }
 
   presence.setActivity(presenceData);
-});
+}
+
+/**
+ * Handle faq page
+ */
+async function handleFAQ(): Promise<void> {
```

Comment:
Why is this async?

Reply:
Futureproofing - there shouldn't be a drawback when using it. I just continued to work with async functions because the main UpdateData-function also uses it.

Reply:
No.